app/models/comment.rb

```diff
@@ -47,8 +47,8 @@ class Comment < ApplicationRecord
   end
 
   after_destroy do
-    self.parent.update_comments_counter
-    participation = author.participations.find_by(target_id: post.id)
+    parent.update_comments_counter if parent
```

Comment:
Style/SafeNavigation: Use safe navigation (`&.`) instead of checking if an object exists before calling the method.

Reply:
I actually agree with @diaspora-code-review here, this is supported since ruby 2.3 and we don't support older versions anymore.